src/System.Drawing.Common/tests/ColorTranslatorTests.cs

```diff
@@ -188,7 +190,16 @@ public static IEnumerable<object[]> FromHtml_TestData()
         [MemberData(nameof(FromHtml_TestData))]
         public void FromHtml_String_ReturnsExpected(string htmlColor, Color expected)
         {
-            Assert.Equal(expected, ColorTranslator.FromHtml(htmlColor));
+            CultureInfo originalCulture = Thread.CurrentThread.CurrentCulture;
```

Comment:
Would it be overkill to add some helper function centrally that takes a callback and a culture it should be run with?